bentoml/cli/bento_service.py

```diff
@@ -96,20 +88,33 @@ def run_with_conda_env(bundle_path, command):
     return
 
 
-def make_bento_name_docker_compatible(name, tag):
+def make_bento_name_docker_compatible(name, version):
     """
     Name components may contain lowercase letters, digits and separators.
     A separator is defined as a period, one or two underscores, or one or more dashes.
 
-    A tag name must be valid ASCII and may contain lowercase and uppercase letters,
-    digits, underscores, periods and dashes. A tag name may not start with a period
+    The version must be valid ASCII and may contain lowercase and uppercase letters,
+    digits, underscores, periods and dashes. Version may not start with a period
     or a dash and may contain a maximum of 128 characters.
 
     https://docs.docker.com/engine/reference/commandline/tag/#extended-description
     """
     name = name.lower().strip("._-")
-    tag = tag.lstrip(".-")[:128]
-    return name, tag
+    version = version.lstrip(".-")[:128]
+    return name, version
+
+
+def validate_tag(ctx, param, tag):  # pylint: disable=unused-argument
+    if ":" in tag:
+        tag = tag.split(":")[:2]
+    else:
+        _echo(
+            f"Tag {tag} does not specify an image version, " f"using 'latest'",
```

Comment:
We should probably not do this either, "latest" is the default for docker, it is also a bad decision on docker side.  And I think the behavior we want, is to use the BentoService version as the default if the user did not specify a version right?